coalib/output/ConsoleInteraction.py

```diff
@@ -855,38 +855,58 @@ def show_enumeration(console_printer,
     console_printer.print()
 
 
+def check_deprecation(show_description,
```

Comment:
this list will grow.  use `**` in the args to collect them all and build a generic reusable algorithm.